etc/helm/pachyderm/templates/_helpers.tpl

```diff
@@ -134,11 +134,15 @@ true
 
 {{- define "pachyderm.userAccessibleOauthIssuerHost" -}}
 {{- if .Values.oidc.userAccessibleOauthIssuerHost -}}
-{{ .Values.oidc.userAccessibleOauthIssuerHost }}
+  {{- if not (hasPrefix "http" .Values.oidc.userAccessibleOauthIssuerHost) -}}
+    {{- printf "%s://%s" (include "pachyderm.hostproto" .) .Values.oidc.userAccessibleOauthIssuerHost -}}
+  {{- else -}}
+  {{ .Values.oidc.userAccessibleOauthIssuerHost }}
+  {{- end -}}
 {{- else if (include "pachyderm.host" .) -}}
-{{- (include "pachyderm.host" .) -}}
+{{- printf "%s://%s" (include "pachyderm.hostproto" .) (include "pachyderm.host" .) -}}
```

Comment:
I don't think you should use this necessarily, but there is actually a URL building function built into helm: https://helm.sh/docs/chart_template_guide/function_list/#urljoin

Reply:
Nice, going to leave it this way now for consistency, but might make sense to switch how we construct must url's to that in a followup PR.